doc/project/priority_definitions.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Generalized Priority Definitions
+---
+
+## Definitions
+
+The priorities described below draw inspiration from the [Google Issue Tracker](https://developers.google.com/issue-tracker/concepts/issues#priority) and have been completely reformulated to fit the OpenTitan setting.
+The following definitions are used:
+
+*  **Work streams**: OpenTitan sub-projects which typically have their own associated working group, e.g.: discrete chip, integrated IP.
+*  **Milestones**: Milestones (M1, M2 etc)  associated with a specific work stream.
+*  **Core function**: Core function of OpenTitan in any of the relevant engineering domains (HW, SW, security).
+
+### Priority Definitions Table
+
+
+<table>
```

Comment:
HTML tables don't render well in mdbook so we might need to convert this. @pamaury for visibility.

Reply:
It's probably better to using Github Markdown tables (they are supported by mdbook). See
[https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables](the doc)

Reply:
Is there a way to render markdown inside a table cell? That is the only reason I added html. I will try to render something with GitHub markdown.

Reply:
I switched to Markdown format. I had to add some partial rows to make it work. Once the new rendering is in place we can revisit this to make it better.